app/ui/src/util/helpText.ts

```diff
@@ -0,0 +1,11 @@
+const PAGE_HELP_BASE_URL = {
+  __DEV_GUIDE__: process.env.REACT_APP_DEV_GUIDE_URL,
```

Comment:
```suggestion
  'https://influxdata.github.io/iot-dev-guide': process.env.REACT_APP_DEV_GUIDE_URL,
```
might be simpler since no changes to existing files are required and they contain valid links that are clickable AS-IS in an editor (such as VS.Code)